pkg/config/value_constraints.go

```diff
@@ -5,15 +5,20 @@ type DistributionVersion struct {
 	Label string `json:"label"` // Static label of the version
 }
 type DistributionArch struct {
-	Name  string `json:"name" `  // Human-readable form of the arch
-	Label string `json:"label" ` //Static label of the arch
+	Name  string `json:"name"`  // Human-readable form of the arch
+	Label string `json:"label"` // Static label of the arch
 }
 
+const ANY_VERSION = "any" //Required
```

Comment:
what does the "Required" comment indicate?

Reply:
I could probably remove that, I suppose at the time it was meant that the other types can change, this one less so.

Reply:
I'm in favor of removing because it confuses me :)